indexing-service/src/test/java/io/druid/indexing/overlord/RemoteTaskRunnerTest.java

```diff
@@ -417,7 +418,9 @@ public boolean isValid()
               {
                 return remoteTaskRunner.getRemovedWorkerCleanups().isEmpty();
               }
-            }
+            },
+            // cleanup task is independently scheduled by event listener. we need to wait some more time.
+            config.getTaskCleanupTimeout().toStandardDuration().getMillis() * 2
```

Comment:
@fjy Sorry, I've confused that two event listeners are on the same path. So it's not related to task cleanup timeout. Though I cannot understand why it takes more than 1 seconds, How about increasing default timeout for `conditionValid` to 2 seconds or more?